### ISSUE TITLE
Netherite Shulker Box particles issue

### ISSUE THREAD

**Issue by asdasdasadg** [OroArmor/Netherite-Plus-Mod#120]
Netherite Shulker Box particles are incorrect.

https://user-images.githubusercontent.com/84078980/120395979-8fa0c980-c33e-11eb-9277-7c3fbda05eb3.png

version: netherite-plus-mod-1.4.5-1.16.5-forge

**Comment by OroArmor** [comment]
Yeah, i dont know how to fix it. Everything seems correct, so its probably something else that i have missed
